nabl2.runtime/trans/nabl2/runtime/transform/renaming.str

```diff
@@ -208,18 +208,19 @@ rules // Utility strategies
     where
       new-name := <show-input-dialog(|"Rename", "")> "Enter a new name"
 
-  get-project-analysis: _ -> analysis-object
+  nabl2-get-project-analysis(multi): ast -> analysis
     where
       project-analyses := <get-project-constraint-analyses>
```

Comment:
I think you could move this operation in the `if` as well.

Reply:
Done